android/src/main/java/com/oney/WebRTCModule/WebRTCModule.java

```diff
@@ -1009,6 +1011,28 @@ public void removeListeners(Integer count) {
 
     private  DailyAudioManager dailyAudioManager;
 
+    @ReactMethod
+    public void setAudioDevice(String deviceId) {
+        this.dailyWebRTCDevicesManager.setAudioDevice(deviceId);
+        // changing the audio mode so Daily won't automatically affect anymore to change the desired audio route
+        if(this.dailyAudioManager != null){
+            this.dailyAudioManager.setMode(DailyAudioManager.Mode.USER_DEFINED);
```

Comment:
hmm...i thought we came up with a different name

Reply:
Rename from USER_DEFINED to USER_SPECIFIED_ROUTE